dist/gcc-compatible/configure

```diff
@@ -194,20 +297,23 @@ fi
 if ! detect_uint128; then
   # Explicitly not supporting compilation with MSVC, which would entail not
   # defining KRML_VERIFIED_UINT128.
-  echo "$CC does not support unsigned __int128 -- using a fallback verified implementation"
+  echo "$CC $CROSS_CFLAGS does not support unsigned __int128 -- using a fallback verified implementation"
   echo "CFLAGS += -DKRML_VERIFIED_UINT128" >> Makefile.config
 fi
 
-if ! detect_ocaml; then
+if [[ "$build_ocaml" == "0" ]] || ! detect_ocaml; then
```

Comment:
nit: a variable named `disable_ocaml` that goes from 0 to 1 is perhaps more intuitive